workflows/gas_nomenclature.nf

```diff
@@ -201,7 +201,9 @@ workflow GAS_NOMENCLATURE {
     clusters = input.filter { meta, file ->
         meta.genomic_address_name != null
     }.collect { meta, file ->
-        meta }
+        meta
+    }.ifEmpty([])
+
```

Comment:
Previously, if all addresses are missing, this would make a null channel and later this null channel would prevent modules from ever executing (append, I think), since Nextflow won't run modules unless all module parameters are non-null. An empty list is non-null, so it will run.